pymc3/model_graph.py

```diff
@@ -107,9 +121,11 @@ def _make_node(self, var_name, graph):
         # Get name for node
         if hasattr(v, 'distribution'):
             distribution = v.distribution.__class__.__name__
+        elif isinstance(v, SharedVariable):
+            distribution = 'Data'
         else:
             distribution = 'Deterministic'
-            attrs['shape'] = 'box'
+        attrs['shape'] = 'box'
```

Comment:
is this making every shape a box?

Reply:
Probably yes.  Looks like I have an indentation error there.  Quick question: when I fix this, should I use a different shape for `Data` than `Deterministic`?  E.g., make `Data` be a roundtangle?

Reply:
OK, here's a fixed version and a sample output graph.
<img width="2080" alt="new-pymc3-graph" src="https://user-images.githubusercontent.com/3274/58433161-742dc480-807b-11e9-8bde-cb8b7c4f00e2.png">

Maybe someone could help me fix the tests so that they check for the proper shapes in an output graph?

Reply:
Tests should be fixed now.

Reply:
What about `Potential` nodes? I think they were triangles back in PyMC2.

Reply:
@fonnesbeck 
> What about `Potential` nodes? I think they were triangles back in PyMC2.

I don't mind having a whack at that, but I'd prefer to have it be a new issue and a new PR.  If you would post a new issue about this, with a small model that has a potential in it, I'll have a look.  But now I am going to squash this commit, and as long as that doesn't break anything, merge it and close this out.